docs/pages/access-controls/idps/saml-reference.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: SAML Identity Provider Reference
+description: Reference documentation for the SAML identity provider
+---
+
+This page provides details on the SAML identity provider built into Teleport.
+
+## What is the SAML identity provider?
+
+The SAML identity provider allows users to use Teleport to authenticate and
+authorize to external applications outside of Teleport that support SAML
+Single Sign-On.
+
+## Basic definitions
+
+The following definitions are paraphrased and summarized from OASIS's SAML
+specification documents, referenced at the bottom of this page.
+
+### Assertions
+
+A package of information that supplies zero or more statements made by an
+identity provider. Assertions are usually made about a subject, which in
+Teleport's case will be Teleport users. These can be viewed as statements of
+fact about a user, e.g. "user *example* belongs to roles *editor*, *access*."
+
+### Service Provider
+
+A service provider can make use of assertions about a subject in order to control
+access and provide customized service. They rely on the **identity provider** for
+these assertions. In Teleport's case, service providers are typically applications
+like Slack, Vercel, etc. that allow SAML Single Sign-On.
+
+### Identity Provider
+
+A service that makes assertions about a subject.
+
+### Service provider initiated SSO
+
+The service provider initiates the login process to an external application.
+
+### Identity provider initiated SSO
+
+The identity provider initiates the login process to an external application.
```

Comment:
These two definitions are identical?

Reply:
They're slightly different, just noting that one is initiated from the identity provider and the other from the service provider. I'll change this up, though, so it's a bit clearer what the difference is.